src/tests/unit/dictionary/skip_list/test_skip_list.c

```diff
@@ -1994,14 +2037,23 @@ void
 runalltests_skiplist(
 ) {
 	/* CuString	*output	= CuStringNew(); */
-	planck_unit_suite_t *suite = skiplist_getsuite();
+	planck_unit_suite_t *suite_1	= skiplist_getsuite_1();
+	planck_unit_suite_t *suite_2	= skiplist_getsuite_2();
+	planck_unit_suite_t *suite_3	= skiplist_getsuite_3();
 
-	planck_unit_run_suite(suite);
 	/* CuSuiteSummary(suite, output); */
 	/* CuSuiteDetails(suite, output); */
 	/* printf("----\nSkiplist Tests:\n%s\n", output->buffer); */
 
-	planck_unit_destroy_suite(suite);
 	/* CuSuiteDelete(suite); */
 	/* CuStringDelete(output); */
+
+	planck_unit_run_suite(suite_1);
```

Comment:
Move the creation of the next suite to be after the destruction of the previous suite - this may help a little bit with memory